raiden/storage/wal.py

```diff
@@ -1,10 +1,15 @@
 import structlog
+import time
 
 from raiden.transfer.architecture import StateManager
 
 log = structlog.get_logger(__name__)  # pylint: disable=invalid-name
 
 
+def _timestamp_ms():
+    return int(time.time() * 1000)
```

Comment:
We don't have a specific use case in mind for this, so why not include the extra information and use milliseconds. We are just adding these timestamps in case some API client should want them in the future - If some raiden gui wants to display internal events being handled it will look cooler with exact times...

Reply:
Why not https://en.wikipedia.org/wiki/ISO_8601 instead of a unix epoch?